### DISCUSSION TITLE
Adding a calendar notification

### DISCUSSION THREAD

**Discussion by ChopsKingsland** [ics-py/ics-py#281] [Q&A] [answered]
Just wondering, is there anyway to automatically add an alarm/reminder/notification to the calendar events?

**Comment by N-Coder** [comment, marked as answer]
See [here](https://github.com/ics-py/ics-py/discussions/277#discussioncomment-288538), which will show you how to add an alarm to a single event. There is no "automated" way to set some alarms by default, but you can simply add the alarm to all events that don't have one set before writing out the calendar:
```python
for event in c.events:
    if not event.alarms:
        event.alarms = [DisplayAlarm(trigger=timedelta(hours=-2))]
with open('my.ics', 'w') as my_file:
    my_file.writelines(c)
```

**Reply by ChopsKingsland** [reply]
ok, how would I add 2 alarms to an event?

**Reply by N-Coder** [reply]
The `alarms` field contains a list of all registered alarms, so:
```python
e.alarms = [DisplayAlarm(trigger=timedelta(hours=-2)), DisplayAlarm(trigger=timedelta(days=-1))]
```

An alternative way of answering "how do I replicate XY with ics.py" is creating an appropriate event with your favourite calendaring GUI, exporting it to ics, and checking which objects ics.py gives you when you parse that file. You can then simply create the same objects programmatically and change them as needed.
